src/Api/Search.php

```diff
@@ -100,14 +102,23 @@ public static function searchForm($args)
             if (is_admin()) {
                 return '';
             }
+            $attsArray = array(
+                'categories_filter_type' => 'select',
+                'pager' => 'simple',
+                'template' => 'grid'
+            );
+
+            $dateSettingsOption = get_option('admwpp_search_settings');
```

Comment:
use the method provided in the Settings class to get the option you need such as "locations_filters"

Reply:
Done